tests/org.jboss.tools.central.ui.bot.test/resources/blacklist-launcher-application-errors-regexes.json

```diff
@@ -0,0 +1,7 @@
+{
+	"LauncherApplicationTest": [
+		".*The element type \"p\" must be terminated by the matching end-tag \"</p>\".*",
+		".*java\.lang\.Object cannot be resolved.*",
+		".*Cannot find the class file for java\.lang\.Object.*"
```

Comment:
@zcervink please remove this java.lan.Object things - as you said, its issue on your local machine related to Java installation